tree-sitter-preproc/bindings/rust/lib.rs

```diff
@@ -0,0 +1,63 @@
+// -*- coding: utf-8 -*-
```

Comment:
Same question here, did you copy this from somewhere, or did you write it from scratch?

Reply:
Same here

Reply:
Done